docs/root/install/tools/route_table_check_tool.rst

```diff
@@ -18,6 +18,10 @@ Input
      :ref:`config <config_tools_router_check_tool>`.
      The tool config input file specifies urls (composed of authorities and paths)
      and expected route parameter values. Additional parameters such as additional headers are optional.
+     Schema: The tool deals with JSON objects internally, based on this :repo:`schema<test/tools/router_check/json/tool_config_schemas.cc>`.
```

Comment:
I would just go ahead and delete the docs about the old schema. Just reword to only reference the new schema.

Reply:
I have a separate change to go after this that uses TCLAP and I am introducing new parameters for the executable. I would wait for the next PR to reword it and put in the TCLAP relevant parameters. Will that work?

Reply:
I would just reword it now. The param documentation is orthogonal.